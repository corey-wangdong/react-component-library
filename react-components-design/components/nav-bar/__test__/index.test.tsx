/*
 * @Author: wangDong Xu
 * @Date: 2022-01-16 10:18:26
 * @LastEditors: wangDong Xu
 * @LastEditTime: 2022-01-16 11:01:33
 * @Description: file content
 * @FilePath: /react-component-library/react-components-design/components/nav-bar/__test__/index.test.tsx
 * @motto: It's not hard to give up, but it must be cool to persist.
 */

import React from 'react';
import renderer from 'react-test-renderer';
import NavBar from '../';

test('NavBar 测试', () => {
  const component = renderer.create(<NavBar title="hello jest" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});