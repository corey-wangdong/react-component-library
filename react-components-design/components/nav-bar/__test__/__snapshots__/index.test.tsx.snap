// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar 测试 1`] = `
<div
  style={
    Object {
      "position": "relative",
    }
  }
>
  <div
    className="navigation"
    style={
      Object {
        "backgroundColor": undefined,
        "paddingTop": undefined,
      }
    }
  >
    <div
      className="navigation__bar"
      style={
        Object {
          "height": undefined,
          "lineHeight": "undefinedpx",
        }
      }
    >
      <div
        className="navigation__bar__left"
        onClick={[Function]}
        style={
          Object {
            "height": undefined,
          }
        }
      >
        <div
          className="navigation__bar__left__back-btn__iocn"
          style={
            Object {
              "backgroundColor": undefined,
            }
          }
        />
      </div>
      <div
        className="navigation__bar__center"
      >
        <span
          className="navigation__bar__center__title"
          style={
            Object {
              "color": undefined,
              "fontSize": undefined,
              "fontWeight": undefined,
            }
          }
        >
          hello jest
        </span>
      </div>
      <div
        className="navigation__bar__right"
        style={
          Object {
            "height": undefined,
          }
        }
      />
    </div>
  </div>
  <div
    style={
      Object {
        "height": undefined,
        "paddingTop": undefined,
      }
    }
  />
</div>
`;
